Code/GraphMol/Wrap/rough_test.py

```diff
@@ -4972,6 +4964,42 @@ def testGitHub1985(self):
     except:
        self.fail("[C@] caused an exception when roundtripping smarts")
 
+  def testGetEnhancedStereo(self):
+
+    rdbase = os.environ['RDBASE']
+    filename = os.path.join(rdbase, 'Code/GraphMol/FileParsers/test_data/two_centers_or.mol')
+    suppl = Chem.SDMolSupplier(filename)
+    m = next(suppl)
+
+    sg = m.GetStereoGroups()
+    self.assertEqual(len(sg), 2)
+    group1 = sg[1]
+    self.assertEqual(group1.GetGroupType(), Chem.StereoGroupType.STEREO_OR)
+    stereo_atoms = group1.GetAtoms()
+    self.assertEqual(len(stereo_atoms), 2)
+    # file is 1 indexed and says 5
+    self.assertEqual(stereo_atoms[1].GetIdx(), 4)
+
+  def testEnhancedStereoPreservesMol(self):
```

Comment:
I don't think this is testing what it's supposed to.
How about this form instead?
```
  def testEnhancedStereoPreservesMol(self):
    """
    Check that the stereo group (and the atoms therein) preserve the lifetime
    of the associated mol.
    """
    rdbase = os.environ['RDBASE']
    filename = os.path.join(rdbase, 'Code/GraphMol/FileParsers/test_data/two_centers_or.mol')
    m = Chem.MolFromMolFile(filename)
    
    sg = m.GetStereoGroups()
    m = None
    self.assertEqual(len(sg), 2)
    group1 = sg[1]
    stereo_atoms = group1.GetAtoms()
    sg = None
    self.assertEqual(stereo_atoms[1].GetIdx(), 4)
    self.assertEqual(stereo_atoms[1].GetOwningMol().GetNumAtoms(),8)
```

Reply:
woah, good catch - my `None`s went away when I was copying the formatting of the other test!

I'll go to:

    stereo_atoms = m.GetStereoGroups()[1].GetAtoms()
    m = None
    gc.collect()
    self.assertEqual(stereo_atoms[1].GetIdx(), 4)
    self.assertEqual(stereo_atoms[1].GetOwningMol().GetNumAtoms(),8) 

the `gc.collect()` is important, because without it there is no guarantee that the Python garbage collector has attempted to reap what it can.